index.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link type="images/png" rel="icon" href="Landing Page/images/logo.png">
+  <link rel="stylesheet"
+    href="https://cdn.jsdelivr.net/npm/@fortawesome/fontawesome-free@6.2.0/css/fontawesome.min.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link
+    href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,200;0,300;0,400;0,500;1,200;1,400&display=swap"
+    rel="stylesheet">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.css"
+    integrity="sha512-5A8nwdMOWrSz20fDsjczgUidUBR8liPYU+WymTZP1lmY9G6Oc7HlZv156XqnsgNUzTyMefFTcsFH/tnJE/+xBg=="
+    crossorigin="anonymous" referrerpolicy="no-referrer" />
+  <link rel="stylesheet" href="style.css">
+  <title>Dream House Online</title>
+</head>
+
+<body>
+  <section class="showcase">
+    <header>
+      <h2 class="logo"></h2>
+      <div class="toggle"></div>
+    </header>
+    <video src="vid01.mp4" muted loop autoplay></video>
+    <div class="ovaerlay"></div>
+
+    <div class="text">
+      <h2 class="title">You're Welcome</h2>
+      <h3>To Dream House Chapel, the God- Kind ministry</h3>
+      <p>In our days, is difficult to distinguish between ministry and industry also known as the God kind. We
+        welcome you once again, join The ministry and you will be blessed. We love you. We celebrate you.<br> GOD
+        BLESS YOU.</p>
+
+      <button class="button-home">
+        <a href="Landing Page/index.html" class="button-home-link">
+          <!-- <h3>Explore Now</h3> -->
+        </a>
+      </button>
+  </section>
+
+
+
+
+
+
+
+
+
+
+
+
+  <!-----
+  <img src="menu.png" id="menuicon">
+  <div class="menu-box" id="menubox">
+    <ul>
+      <li><a href="Landing Page/index.html">Home</a></li>
+      <li><a href="Landing Page/Service.html">Service</a></li>
+      <li><a href="#">Lives</a></li>
+      <li><a href="Landing Page/about.html">About</a></li>
+      <li><a href="Landing Page/contact.html">Contact</a></li>
+    </ul>
+
+    <script>
+      let menubox = document.getElementById("menubox")
+      let menuicon = document.getElementById("menuicon")
+
```

Comment:
Also, it is not a good practice to leave commented codes in a project. You can check out [this](https://www.w3schools.com/js/js_best_practices.asp) article to understand it more. 

Please delete all unused codes.